test/Manifest.toml

```diff
@@ -0,0 +1,204 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
This file should no be here either.
I recommend creating a global gitignore and adding `Manifest.toml` to it.